elements/pfe-icon/package.json

```diff
@@ -9,33 +9,24 @@
     "access": "public"
   },
   "description": "Icon element for PatternFly Elements",
-  "keywords": [
```

Comment:
Why remove the keywords?

Reply:
It's a brand new element; I'll pull in the keywords from the previous package.json.

Reply:
Sorry, it's confusing with the diffs showing up the way they are.  I'll remember that for next time!